apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -337,10 +337,10 @@ export const include: Includeable[] = [
 
 export const order: OrderItem[] = [
   [{ model: Defendant, as: 'defendants' }, 'created', 'ASC'],
+  [{ model: CivilClaimant, as: 'civilClaimants' }, 'created', 'ASC'],
```

Comment:
Is it because of alphabetical order of models for ASC and then DESC?

Reply:
I did this for consistency, but this is totally unnecessary. The CivilClaimant model is included in Case right after the Defendant model so I thought sorting using the same order made sence. Just some OCD kicking in I guess.